### ISSUE TITLE
Yomichan icon is displayed with a red box of exclamation points

### ISSUE THREAD

**Issue by Ceiyne** [FooSoft/yomichan#713]
In the past couple of weeks I've noticed that from time to time the Yomichan icon in the Google Chrome toolbar will display a red box with two exclamation points in it, like this:
![image](https://user-images.githubusercontent.com/20176747/89304311-4c50ce00-d63b-11ea-8c2a-31a754279270.png)
If I mouse over it, the pop-up displays:

Yomichan - Error
Has access to this site

If I click the Yomichan button, the red box goes away and the Yomichan icon returns to normal, and works fine.  I'm not 100% sure, but I think that Yomichan is non-functional (no pop-up when mousing over Japanese text) while the red box is shown.  I'll have to try to test that next time it occurs.

I haven't noticed a specific pattern of when this occurs.  I've only noticed it on my Windows PC and not on my Chromebook, but I use Yomichan more on my PC so it's possible it's happening in both places.

Is there a log or something else I could check for more information to send you?

**Comment by toasted-nutbread** [comment]
Yeah, the "!!" typically indicates that an error occurred at some point. If the browser hasn't been restarted, the logs can be viewed as follows:

* Chrome:
  * Open the browser's extensions page (`chrome://extension`):
  * Enabling "Developer mode" on the top right,
  * Where it says "Inspect views", click the "bg/background.html" link (may have to view the "more..." link first).
  * Open the "Console" tab in the window that appears.
  * Errors should be listed here.

* Firefox:
  * Open `about:devtools-toolbox?type=extension&id=alex%40foosoft.net`.
  * Open the "Console" tab.
  * Errors should be listed here.

The positive is that these issues are typically not indicative of the extension being broken, but rather that something went wrong at _some_ point and was recoverable. These things are hard to test for because they usually involve bizarre edge cases or browser bugs, so feel free to share any logs when you run into these issues. Longer term, I plan to add a more user-friendly method of viewing these logs.

**Comment by Ceiyne** [comment]
This issue didn't occur for a long time, but I had it today.  Here's what I got out of the dev tools:
![image](https://user-images.githubusercontent.com/20176747/90245392-95094380-de00-11ea-9e30-f1f1f1111533.png)

I wasn't trying to use Yomichan on the site (xfinity.com) mentioned in the logs (but of course, it was enabled in the browser).  While the red exclamation points were displayed, I was able to continue using Yomichan normally in other tabs.

**Comment by toasted-nutbread** [comment]
Yeah, I see what would be happening with that. Depending on how a SVG image is embedded into a webpage, it may be displayed as a full document, similar to an `<iframe>`. However, it's not the same as a standard HTML document, so things go wrong. I'll look into handling that as a special case.

**Comment by Ceiyne** [comment]
Thanks.  No big deal for me, though. I just wanted to report it.

**Comment by xerpi** [comment]
I have the same issue, in case it helps:
![image](https://user-images.githubusercontent.com/560596/90369824-aeaec300-e06c-11ea-8600-cf9582ad5b7b.png)
